fuzzypp/test/preprocessor_test.cpp

```diff
@@ -0,0 +1,332 @@
+#include <catch.hpp>
```

Comment:
General question concerning behaviour and tests: what's our expecation for macros? For instance, would fuzzyc2cpg interpret function- and object-like macros as is? If they must be expanded, I'd assume that **simplecpp** itself has tests for those, right? Does it handle [self referential macros](https://gcc.gnu.org/onlinedocs/gcc-3.3.6/cpp/Self_002dReferential-Macros.html) well?

Reply:
Yes, I believe it already handles these cases. It won't stop you from infinitely recursing, though :wink: